coverage/sqldata.py

```diff
@@ -1133,6 +1133,11 @@ def _connect(self) -> None:
         # PyPy needs the .close() calls here, or sqlite gets twisted up:
         # https://bitbucket.org/pypy/pypy/issues/2872/default-isolation-mode-is-different-on
         self.execute_void("pragma journal_mode=off")
+        if self.execute_one("pragma journal_mode") != "off":
```

Comment:
When the setting is successful, the value returned is `("off",)`, not `"off"`.

Reply:
Good catch. I'll verify that on my end and fix it.